source/guides/clear/compatible-kernels.rst

```diff
@@ -12,7 +12,7 @@ Bare metal only
 
 Kernel native
     The *kernel-native* bundle focuses on the bare metal platforms. It is
-    optimized for fast booting and performs best on the Intel® architectures
+    optimized for fast booting and performs best on the Intel processors
```

Comment:
Correct this per the DB.  If you disagree, show me the example where the phrase "Intel processor" originates.  
Intel® Architecture Processors